src/command.rs

```diff
@@ -1000,7 +998,11 @@ impl CommandBuffer {
             return Err(CommandError::TYPE_ERR);
         }
 
-        if !(RowMatrix::new(affine.transformation).det().abs() >= f32::EPSILON) {
+        if let Some(Ordering::Less) = RowMatrix::new(affine.transformation)
+            .det()
+            .abs()
+            .partial_cmp(&f32::EPSILON)
+        {
```

Comment:
:nauseated_face: Also not the same. We want `NaN` to fail.

Reply:
Next time I should just use the automatic rewriting... :robot: